src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx

```diff
@@ -65,7 +65,7 @@ export const MarketingLandingApp = () => {
           {/* <MeetTheSpecialists /> */}
           <CollectorsOverview />
           <PreviouslySoldOnArtsyRailQueryRenderer />
-          <Reviews />
+          {/* <Reviews /> */}
```

Comment:
Same here.

Reply:
Will remove the comment, valid point anyways, thanks

Reply:
Oh, I see. I would try to avoid such comments whenever possible. But it's up to you to leave hints here to point out that this component already exists :)

Reply:
Left the hints :)

Reply:
Feel free to merge 👍